### PR TITLE
add program page

### PR DESCRIPTION
**Pull request by bbrcknl**
Signed-off-by: Birgit Brecknell <birgit@sel4.systems>

### CODE REVIEW

File: content/Foundation/Summit/program.pml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <th colspan="2">Day 2&emsp;11 October 2022</th>
+  </tr>
+  <tr><td>9:00 - 9:50</td><td><strong>Keynote: Kry10 Secure Platform</strong><br>Boyd Multerer, Kry10</td></tr>
+  <tr><td>9:50 - 10:00</td><td><em>(videos and/or prez of summit 23)</em></td></tr>
+  <tr><td>10:15 - 10:45</td><td><strong>seL4 and BEAM: a match made in Erlang</strong><br>Ihor Kuz, Kry10</td></tr>
+  <tr><td>10:45 - 11:15</td><td><strong>Security framework for mixed-critical use-cases</strong><br>Jorge Pereira</td></tr>
+  <tr><td>11:15 - 11:45</td><td><strong>Rust support in seL4 userspace: Present and future lightning talk + discussion</strong><br>Nick Spinale</td></tr>
+  <tr><td>12:45 - 13:15</td><td><strong>Trustworthy Board Management Controllers</strong><br>Daniel Schwyn & Ben Fiedler, ETH</td></tr>
+  <tr><td>13:15 - 13:45</td><td><strong>Early Experiences Proving the Correctness of a Network Stack Implementation</strong><br>Alain Kägi et al, Lewis & Clark College</td></tr>
+  <tr><td>13:45 - 14:15</td><td><strong>CASE overviewCyber Assured Systems Engineering</strong><br>Darren Cofer, Collins Aerospace</td></tr>
```
Comment (axel-h) [inline content/Foundation/Summit/program.pml:44]:
```suggestion
  <tr><td>13:45 - 14:15</td><td><strong>CASE overview: Cyber Assured Systems Engineering</strong><br>Darren Cofer, Collins Aerospace</td></tr>
```

**Comment by june-andronick** [comment]
The linker issue seems to be a time out (from Matt's website)

**Review by gernotheiser — APPROVED** [review]
Looks mostly good, but two questions:

1. June, Gerwin and myself are sometimes listed as seL4 Foundation, sometimes with their home organisation. Is this intended?
2. Some titles use title case, others only capitalise the first word. Should be more consistent

**Comment by june-andronick** [comment]
> Looks mostly good, but two questions:
> 
> 1. June, Gerwin and myself are sometimes listed as seL4 Foundation, sometimes with their home organisation. Is this intended?

Yes. The ones with the seL4 Foundation affiliations are more on behalf of the Foundation (the overviews, the Foundation update), whereas the others are on work in our home organisations.

> 2. Some titles use title case, others only capitalise the first word. Should be more consistent

Thanks for spotting this. Should be fixed now.

**Pull request merged**